src/panels/config/voice-assistants/entity-voice-settings.ts

```diff
@@ -148,78 +155,86 @@ export class EntityVoiceSettings extends SubscribeMixin(LitElement) {
-                (googleManual && key === "cloud.google_assistant")}
-                .checked=${alexaManual && key === "cloud.alexa"
-                  ? manExposedAlexa
-                  : googleManual && key === "cloud.google_assistant"
-                  ? manExposedGoogle
-                  : this.entry.options?.[key]?.should_expose}
-              ></ha-switch>
-            </ha-settings-row>`
+            (key) => html`
+              <ha-settings-row>
+                <img
+                  alt=""
+                  src=${brandsUrl({
+                    domain: voiceAssistants[key].domain,
+                    type: "icon",
+                    darkOptimized: this.hass.themes?.darkMode,
+                  })}
+                  referrerpolicy="no-referrer"
+                  slot="prefix"
+                />
+                <span slot="heading">${voiceAssistants[key].name}</span>
+                ${key === "cloud.google_assistant" &&
+                !googleManual &&
+                this._googleEntity?.might_2fa
+                  ? html`
+                      <ha-formfield
+                        slot="description"
+                        .label=${this.hass.localize(
+                          "ui.dialogs.voice-settings.ask_pin"
+                        )}
+                      >
+                        <ha-checkbox
+                          .checked=${!this.entry.options?.[key]?.disable_2fa}
+                          @change=${this._2faChanged}
+                        ></ha-checkbox>
+                      </ha-formfield>
+                    `
+                  : (alexaManual && key === "cloud.alexa") ||
+                    (googleManual && key === "cloud.google_assistant")
+                  ? html`
+                      <span slot="description">
+                        ${this.hass.localize(
+                          "ui.dialogs.voice-settings.manual_config"
+                        )}
+                      </span>
+                    `
+                  : nothing}
+                <ha-switch
+                  .assistant=${key}
+                  @change=${this._toggleAssistant}
+                  .disabled=${(alexaManual && key === "cloud.alexa") ||
+                  (googleManual && key === "cloud.google_assistant")}
+                  .checked=${alexaManual && key === "cloud.alexa"
+                    ? manExposedAlexa
+                    : googleManual && key === "cloud.google_assistant"
+                    ? manExposedGoogle
+                    : this.entry.options?.[key]?.should_expose}
+                ></ha-switch>
+              </ha-settings-row>
+            `
           )
-        : ""}
+        : nothing}
+
+      <h3>${this.hass.localize("ui.dialogs.voice-settings.aliases_header")}</h3>
 
-      <h3>
-        ${this.hass.localize("ui.dialogs.voice-settings.aliasses_header")}
-      </h3>
+      <p>
+        ${this.hass.localize("ui.dialogs.voice-settings.aliases_description")}
```

Comment:
Can we make this secondary text color?

Reply:
And less space between the header and text?

![CleanShot 2023-04-19 at 10 02 49](https://user-images.githubusercontent.com/5662298/233009392-6da7afcb-ed4e-41d5-8d85-1435b4658741.png)